src/server/lib/middleware/helmet.ts

```diff
@@ -35,6 +36,7 @@ const helmetConfig = {
         gaUID.hash, // google analytics id
         CSP_VALID_URI.GOOGLE_ANALYTICS,
         CSP_VALID_URI.CMP,
+        ...(stage === 'DEV' ? [HELMET_OPTIONS.UNSAFE_EVAL] : []),
```

Comment:
This ternary spread operator combo is a bit narly and it's in an area that is arguably critically important. Shall we break this logic out so that we define the `scriptSrc` array above the `helmetConfig` object? That was we can use much simpler javascript for conditionally adding to the array Eg `if(stage === 'DEV') scriptSrc.push(HELMET_OPTIONS.UNSAFE_EVAL)` which is a lot easier to read